src/lib/utils.ts

```diff
@@ -106,20 +106,24 @@ export const squashReportByFileExtension = (report: IFileSizeReport): IFileSizeR
  * @param report
  */
 export const normalizeSlugsInFileNames = (report: IFileSizeReport): IFileSizeReport => {
-  const ELLIPSIS = '.[…].';
-
   const normalizedKeys = Object.keys(report).map((fileName: string) => {
-    const fileNameSlugs = path.parse(fileName).name.split('.');
-    const acceptancePredicates: Array<(_: string[]) => boolean> = [
-      // Are there at least 2 slugs?
-      _ => _.length > 1,
-      // Is the last slug different from `min`?
-      _ => _.slice(-1)[0] !== 'min'
-    ];
-
-    return acceptancePredicates.reduce((seq: boolean, fn) => seq && fn(fileNameSlugs), true)
-      ? replace(`.${fileNameSlugs[fileNameSlugs.length - 1]}.`, ELLIPSIS, fileName)
-      : fileName;
+    const { base, dir } = path.parse(fileName);
+    const fileNameSlugs = base.split('.');
+
+    const normalizedFileName = fileNameSlugs
+      .map((slug, index) => {
+        switch (true) {
+          case index === 0:
+          case index === fileNameSlugs.length - 1:
```

Comment:
noticing that CRA (create-react-app) deploys webpack with config

```
static/js/[name].[contenthash:8].chunk.js
```

maybe is worth to add an exception for this famous deployed config as well

Reply:
Done, we're now keeping `.chunk` in the report filename